compose/cli/errors.py

```diff
@@ -40,10 +40,10 @@ def __init__(self):
         """)
 
 
-class ConnectionErrorBoot2Docker(UserError):
+class ConnectionErrorDockerMachine(UserError):
     def __init__(self):
-        super(ConnectionErrorBoot2Docker, self).__init__("""
-        Couldn't connect to Docker daemon - you might need to run `boot2docker up`.
+        super(ConnectionErrorDockerMachine, self).__init__("""
+        Couldn't connect to Docker daemon - you might need to run `docker-machine start default`.
```

Comment:
Where does `default` come from?  It looks like the machine docs use `dev` (https://docs.docker.com/machine/get-started/).

Is this the machine name that toolbox uses?